src/linear_algebra/bilinear_form.lean

```diff
@@ -948,39 +948,47 @@ end to_matrix
 
 end matrix
 
-namespace refl_bilin_form
-
-open refl_bilin_form bilin_form
+namespace bilin_form
 
 /-- The proposition that a bilinear form is reflexive -/
 def is_refl (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = 0 → B y x = 0
 
-variable (H : is_refl B)
+end bilin_form
+
+namespace bilin_form.is_refl
+
+open bilin_form.is_refl bilin_form
+
+variable (H : B.is_refl)
 
 lemma eq_zero : ∀ {x y : M}, B x y = 0 → B y x = 0 := λ x y, H x y
 
 lemma ortho_sym {x y : M} :
   is_ortho B x y ↔ is_ortho B y x := ⟨eq_zero H, eq_zero H⟩
 
-end refl_bilin_form
-
-namespace sym_bilin_form
+end bilin_form.is_refl
```

Comment:
```suggestion
end is_refl
```

Reply:
Ah thanks, I did not know that. I should really read Theorem Proving in Lean..